### PR TITLE
Api - initial bogus api

### PR DESCRIPTION
**Pull request by HansDahle**

### CODE REVIEW

File: src/backend/api/Fusion.Resources.Api/Controllers/Personnel/PersonnelController.cs [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                JobTitle = request.JobTitle,
+                Mail = request.Mail,
+                PhoneNumber = request.PhoneNumber
+            };
+            
+            return Created($"/projects/{projectIdentifier}/contracts/{contractIdentifier}/resources/personnel/{item.Mail}", item);
+        }
+
+        [HttpPost("/projects/{projectIdentifier}/contracts/{contractIdentifier}/resources/personnel-collection")]
+        public async Task<ActionResult<ApiBatchResponse<ApiContractPersonnel>>> CreateContractPersonnelBatch(string projectIdentifier, string contractIdentifier, [FromBody] IEnumerable<CreateContractPersonnelRequest> requests)
```
Comment (MartinForre) [inline src/backend/api/Fusion.Resources.Api/Controllers/Personnel/PersonnelController.cs]: Does dotnet mvc handle `[FromBody] IEnumerable<T>`? I think that I've had to wrap enumerables in another type before 🤔

File: src/backend/api/Fusion.Resources.Api/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+            #region Health probes
+
+            app.UseHealthChecks("/_health/liveness", new HealthCheckOptions
+            {
+                Predicate = r => r.Name.Contains("liveness")
+            });
+            app.UseHealthChecks("/_health/ready", new HealthCheckOptions
+            {
+                Predicate = r => r.Name.Contains("liveness")
```
Comment (MartinForre) [inline src/backend/api/Fusion.Resources.Api/Startup.cs]: Will `r.Name` contain `"liveness"` in this case? Shouldn't it be `"ready"`?

**Comment by HansDahle** [comment]
Hmm, did not test 😃 time will tell..

File: src/backend/api/Fusion.Resources.Api/Controllers/Contracts/ContractsController.cs [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+        [HttpPost("/projects/{projectIdentifier}/contracts/{contractIdentifier}/external-contract-responsible")]
+        public async Task<ActionResult> CreateContractExternalContractResp(string projectIdentifier, string contractIdentifier, [FromBody] ContractPositionRequest request)
+        {
+
+            return Ok();
+        }
+    }
+
+    public class ContractPositionRequest
```
Comment (MartinForre) [inline src/backend/api/Fusion.Resources.Api/Controllers/Contracts/ContractsController.cs]: Move all these models to seperate files?

**Comment by HansDahle** [comment]
The liveness refers to the test named in the AddHealthCheck in configure services.

File: src/backend/api/Fusion.Resources.Api/Startup.cs (hunk 2 above)
Comment (HansDahle) [reply src/backend/api/Fusion.Resources.Api/Startup.cs]: that predicate defined which tests, defined in the configure services section, to execute for that endpoint.

File: src/backend/api/Fusion.Resources.Api/Controllers/Personnel/PersonnelController.cs (hunk 1 above)
Comment (HansDahle) [reply src/backend/api/Fusion.Resources.Api/Controllers/Personnel/PersonnelController.cs]: didn't test, will find out :) anyway just initial setup.

File: src/backend/api/Fusion.Resources.Api/Controllers/Contracts/ContractsController.cs (hunk 3 above)
Comment (HansDahle) [reply src/backend/api/Fusion.Resources.Api/Controllers/Contracts/ContractsController.cs]: yeh, will do when progressing a bit.. suspects a some changes are gonne come :)

**Pull request merged**